tweet.md

```diff
@@ -48,6 +48,8 @@ var twitterHandle = '@tesselproject';
 var status = 'Hello ' + twitterHandle + '. This is your #Tessel 2 speaking.';
 
 // Enter the oauth key and secret information
+// These OAuth credentials are for the dummy @TesselTweet account
```

Comment:
This & the next line are good, maybe that's all it needs? (As in, can we cut the new paragraph above?)